test/extension.test.ts

```diff
@@ -557,4 +568,33 @@ suite('Compilation info', () => {
     expect(info.compileFlags[0]).to.eq('/Z+:some-compile-flag');
     expect(info.compiler).to.eq('cl.exe');
   });
+});
+
+suite('CMake System tests', () => {
+  (process.env.HasVs == 'true' ? suite.skip : suite)('No present kit',() => {
+        suiteSetup(()=>{
+          clearExistingKitConfigurationFile();
+          process.env.PATH = "";
```

Comment:
Should the PATH Be reset once we're done with this test?

Reply:
You are right, the clean up part was missing.